pkg/api/types.go

```diff
@@ -805,6 +805,19 @@ func (config TestStepConfiguration) IsPeriodic() bool {
 	return config.Interval != nil || config.MinimumInterval != nil || config.Cron != nil || config.ReleaseController
 }
 
+// GetClusterProfileName returns the cluster profile name if it's set
+func (config TestStepConfiguration) GetClusterProfileName() string {
+	switch {
+	case config.MultiStageTestConfigurationLiteral != nil:
+		return config.MultiStageTestConfigurationLiteral.ClusterProfile.Name()
+	case config.MultiStageTestConfiguration != nil:
+		return config.MultiStageTestConfiguration.ClusterProfile.Name()
+	case config.OpenshiftInstallerClusterTestConfiguration != nil:
+		return config.OpenshiftInstallerClusterTestConfiguration.ClusterProfile.Name()
+	}
+	return ""
```

Comment:
you can add the `default` in the switch and you don\t have to return.. However, is the default empty string a valid case?

Reply:
Ok. And yeah empty string is valid, I deal with it in another function. Only jobs which have cluster profiles defined have the value something other than the empty string.